doc/quick-start-user-guide.md

```diff
@@ -71,9 +77,41 @@ etcdcurls = "http://192.168.56.101:2379"
 etcdpurls = "http://192.168.56.101:2380"
 ```
 
+**Authentication:** In glusterd2 REST API authentication is enabled by default. To disable rest authentication add `restauth=false` in Glusterd2 config file(Default path is
+`/var/lib/glusterd2/glusterd2.toml` in case of rpm installation) or in the config file provided by you (conf.toml as per above example)
+
 Replace the IP address accordingly on each node.
 
-**Start glusterd2 process:** Glusterd2 is not a daemon and currently can run only in the foreground.
+### Running glusterd2 for RPM installation
+
+**Enable glusterd2 service:** To enable glusterd2 service run:
```

Comment:
Enabling can mean anything. Please mention that this command is to ensure glusterd2 service starts automatically when the system starts